### PR TITLE
Add bot to #jenkins-community

### PR DESCRIPTION
**Pull request by rtyler**
I think I'm doing this right o_O

### CODE REVIEW

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotConfig.java [hunk 1]
```diff
@@ -14,37 +14,37 @@
  * @author Oleg Nenashev <o.v.nenashev@gmail.com>
  */
 public class IrcBotConfig {
-    
+
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcBotConfig.java]: Refactoring is not a topic for this PR

**Comment by oleg-nenashev** [comment]
Should work. jenkins-meeting may be also useful to have the automatic referencing of JIRA issues there

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotConfig.java (hunk 1 above)
Comment (rtyler) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcBotConfig.java]: IMHO removing whitespace isn't refactoring by definition. But I also believe in leaving source code cleaner than you found it. If you do not want this commit I can remove it.

**Comment by orrc** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
:+1: 
Merging, leaving the deployment to @rtyler

**Pull request merged**